packages/analytics/amplify_analytics_pinpoint/example/integration_test/identify_user_test.dart

```diff
@@ -105,6 +107,16 @@ void main() {
                   ),
                 )
                 .having((e) => e.endpoint.user?.userId, 'UserId', userId)
+                .having(
+                  (e) => e.endpoint.user?.userAttributes?.toMap() ?? const {},
+                  'Attributes',
```

Comment:
```suggestion
                  'UserAttributes',
```

Reply:
Although this is confusing - why does `Attributes` below have `name` and `email`, but then these "user attributes" have random values. Shouldn't these be things like phone number, birthday, etc.?

Reply:
It's because Analytics organization is a bit strange now ... Abdallah says we'll be revamping Analytics particularly because of these legacy behaviors that shouldn't be supported into the future :) 

We call this code: 

```
await Amplify.Analytics.identifyUser(
          userId: userId,
          userProfile: AWSPinpointUserProfile(
            name: name,
            email: email,
            plan: plan,
            location: location,
            analyticsProperties: properties,
            userAttributes: properties,
          ),
        );
```

The AwsPinpointUserProfile name, email, plan fields are mapped into Pinpoint Endpoint attributes. 

However, the AwsPinpointUserProfile userAttributes are mapped into Pinpoint Endpoint user attributes. 

Thus the integration test checks for that, but if you see the triggering API call it doesn't match at all.  It's great how integration tests can show how a public API isn't good, when the code expectation doesn't match what's actually being verified.